test/test_fx_backends.py

```diff
@@ -196,10 +196,6 @@ def fn(x):
         nvfuser = NvFuserBackend()
         compiled_module = nvfuser.compile(copy.deepcopy(traced))
 
-        for node in compiled_module.graph.nodes:
-            if node.op == "call_function":
-                assert "fused" in str(node.target), "the entire function should be fused into a single fusion group"
```

Comment:
dumb question, I thought we have fn composed fully of nvfuser compatible nodes, is there any reason that we are not getting the full graph fused?

We should still try to check that portion of the graph has been fused, otherwise, this could end up being a false positive test.

Reply:
Now what's returned by `nvfuser.compile` is a copy of the input graph module with a call function modified which lowers to prims and executes the partially lowered graph.

I agree that it would be useful to access this lowered graph that's being executed. I will reorganize the code a bit and add those tests for checking the graph nodes.